builders/typescript.build

```diff
@@ -1,4 +1,4 @@
 {
-    "cmd": ["tsc", "--out", "${file/\\.ts/\\.js/}", "$file"],
+    "cmd": ["tsc", "--out", "${file\\.ts\\.js/}", "$file"],
```

Comment:
@bradydowling shouldn't the --out parameter be removed as done in c9/core#26 (comment)?